client/app/components/Link.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
```

Comment:
I know this file was already structured this way, but this feels pretty confusing, right? Not for now, but maybe later we can refactor this? Or am I missing something?

Reply:
It does, but we cannot do it for now because this is a compatibility feature for Managed Redash